documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -6,8 +6,9 @@
 = Secrets
 
 [role="_abstract"]
-Strimzi uses secrets to store private and public key certificates for Kafka clusters, clients, and users.
-Secrets are used for establishing TLS encrypted connections between Kafka brokers, and between brokers and clients.
+Secrets are created when Strimzi is installed. 
```

Comment:
My view ... Strimzi installation => operator installation ... but that does not create any secrets. Only when you deploy some custom resource something happens.

```suggestion
Secrets are created when Kafka cluster is deployed. 
```